brain/validator/validate_password.py

```diff
@@ -0,0 +1,130 @@
+'''@validate_password
+
+This file contains functions for validating the user's password against
+a set of requirements.
+
+'''
+
+from flask import current_app
+import yaml
+
+
+def load_min(app=True, root='/vagrant'):
+    '''@load_min
+
+    This method returns the minimum character requirement for passwords.
+
+    @app, indicates if function is to be used by application, or manually
+
+    '''
+    if app:
+        min_c = current_app.config.get('PASSWORD_MIN_C')
+        return {'password_min_c': min_c, 'error': None}
+    else:
+        with open(root + "/hiera/settings.yaml", 'r') as stream:
+            try:
+                min_c = \
+                       yaml.load(stream)['validate_password']['password_min_c']
+                return {'password_min_c': min_c, 'error': None}
+            except yaml.YAMLError as error:
+                return {'password_min_c': None, 'error': error}
+
+
+def load_max(app=True, root='/vagrant'):
+    '''@load_max
+
+    This method returns the maximum character requirement for passwords.
+
+    @app, indicates if function is to be used by application, or manually
+
+    '''
+    if app:
+        max_c = current_app.config.get('PASSWORD_MAX_C')

```

Comment:
I would change `current_app.config.get('PASSWORD_MIN_C')` to `current_app.config.get('PASSWORD_MAX_C', None)`.  This way, instead of logic raising an `KeyError`, it will default to `None` if no keys was found via `get`.